unison-share-projects-api/src/Unison/Share/API/Hash.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
We should probably fix these JWT typeclass deprecations, but I'm not totally sure I understand how.

Reply:
Okay. Maybe could just capture the deprecation warnings and post them in a new ticket?

Reply:
Yeah, agreed with the ticket thing – also add a comment like

```suggestion
-- TODO: Fix the JWT deprecations that trigger this (see #1234)
{-# OPTIONS_GHC -Wno-deprecations #-}
```
so that someone who wonders why it’s there can discover the context.